### PR TITLE
Changed a lot of things that CHARL deleted.

### PR DESCRIPTION
**Pull request by ScheepersDev23**

### CODE REVIEW

File: src/Food.cpp [hunk 1]
```diff
@@ -19,3 +27,7 @@ const std::vector<int> &Food::getPosition() const {
 void Food::setPosition(const std::vector<int> &position) {
     Food::position = position;
 }
+
+Food::~Food() {
+    std::cout << "Food Destructor Called" << std::endl;
+}
```
Comment (charl26) [inline src/Food.cpp]: Add a new line at end of file

File: src/main.cpp [hunk 2]
```diff
@@ -45,23 +46,20 @@ void startGame() {
...
+            }
+        } else {
+            std::cout << "ERROR : Window Width is incorrect." << std::endl;
+        }
+    } else {
 		std::cout << "ERROR : Incorrect amount of arguments." << std::endl;
 	}
+    // TODO: These nested IF's are messy. Clean it up.
 	return 0;
 }
```
Comment (charl26) [inline src/main.cpp]: Add new line at end of file

File: src/main.cpp [hunk 3]
```diff
@@ -45,23 +46,20 @@ void startGame() {
...
+            } else {
+                std::cout << "ERROR : Window Height is incorrect." << std::endl;
+            }
+        } else {
+            std::cout << "ERROR : Window Width is incorrect." << std::endl;
+        }
+    } else {
 		std::cout << "ERROR : Incorrect amount of arguments." << std::endl;
 	}
+    // TODO: These nested IF's are messy. Clean it up.
```
Comment (charl26) [inline src/main.cpp]: Complete TODO

File: src/snake.cpp [hunk 4]
```diff
@@ -4,10 +4,39 @@
...
+	Snake::head = rhs.isHead();
+
+	return (*this);
+}
+
+Snake::Snake(const Snake &snake) {
+	Snake::operator=(snake);
+}
+
+Snake::Snake() {}
```
Comment (charl26) [inline src/snake.cpp]: ADD new line

**Review by charl26 — CHANGES_REQUESTED** [review]
FIX TODO

**Review by charl26 — APPROVED** [review]

**Review by koekemoer — APPROVED** [review]

**Pull request merged**